ush/config.sh.RRFS_CONUS_13km

```diff
@@ -46,7 +46,8 @@ if [[ ${DO_ENSEMBLE}  == "TRUE" ]]; then
    NUM_ENS_MEMBERS=9
    DO_SPPT=TRUE
    SPPT_MAG=0.5
-   DO_DACYCLE="TRUE"
+   DO_DACYCLE="FALSE"
+   DO_ENKFUPDATE="TRUE"
```

Comment:
Any others use the file "config.sh.RRFS_CONUS_13km"?  If yes, we may want to keep DO_DACYCLE="TRUE" and DO_ENKFUPDATE="FALSE" by default?

Reply:
They are set this way only for ensemble run (DO_ENSEMBLE=TRUE). The default value is DO_DACYCLE="TRUE" (earlier section of this config file) and DO_ENKFUPDATE="FALSE" (default value)
Thanks!